### PR TITLE
Update dependencies 

### PR DESCRIPTION
**Pull request by vdemeester**
Depends on https://github.com/docker/app/pull/244

- Bump kubernetes to 1.8.14
- Bump other dependencies to more recent version (go-winio got a fix, etc..)

Signed-off-by: Vincent Demeester <vincent@sbr.pm>

### CODE REVIEW

File: internal/renderer/render.go [hunk 1]
```diff
@@ -218,11 +232,59 @@ func Render(appname string, composeFiles []string, settingsFile []string, env ma
...
+
+func substitute(template string, mapping composetemplate.Mapping) (string, error) {
+	return composetemplate.SubstituteWith(template, mapping, pattern, errorIfMissing)
+}
+
+func errorIfMissing(substitution string, mapping composetemplate.Mapping) (string, bool, error) {
+	value, found := mapping(substitution)
+	if !found {
+		return "", true, &composetemplate.InvalidTemplateError{
+			Template: fmt.Sprint("required variable" + substitution + "is missing a value"),
```
Comment (silvin-lubecki) [inline internal/renderer/render.go]: nit: use `fmt.Sprintf` instead?

**Comment by silvin-lubecki** [comment]
⚠️ Please rebase 🙊

**Comment by vdemeester** [comment]
@silvin-lubecki rebased :hear_no_evil:

File: Jenkinsfile [hunk 2]
```diff
@@ -69,6 +69,8 @@ pipeline {
                         dir('src/github.com/docker/app') {
                             checkout scm
                             sh 'make -f docker.Makefile coverage'
+                            archiveArtifacts '_build/ci-cov/all.out'
+                            archiveArtifacts '_build/ci-cov/coverage.html'
```
Comment (mat007) [inline Jenkinsfile]: Why do we need them now and not before?

Comment (vdemeester) [reply Jenkinsfile]: @mat007 this is _sadly_ what triggers a build fail or not (i.e. if the artifacts are here or not)

**Pull request merged**